gateway/ln-gateway/src/lightning/ldk.rs

```diff
@@ -225,45 +215,15 @@ impl Drop for GatewayLdkClient {
 impl ILnRpcClient for GatewayLdkClient {
     async fn info(&self) -> Result<GetNodeInfoResponse, LightningRpcError> {
         let node_status = self.node.status();
-
-        let Some(chain_tip_block_summary) = self
-            .esplora_client
-            .get_blocks(None)
-            .await
-            .map_err(|e| LightningRpcError::FailedToGetNodeInfo {
-                failure_reason: format!("Failed get chain tip block summary: {e:?}"),
-            })?
-            .into_iter()
-            .next()
-        else {
-            return Err(LightningRpcError::FailedToGetNodeInfo {
-                failure_reason:
-                    "Failed to get chain tip block summary (empty block list was returned)"
-                        .to_string(),
-            });
-        };
-
-        let esplora_chain_tip_timestamp = chain_tip_block_summary.time.timestamp;
-        let block_height: u32 = chain_tip_block_summary.time.height;
-
-        let synced_to_chain = node_status
-            .latest_lightning_wallet_sync_timestamp
-            .unwrap_or_default()
-            > esplora_chain_tip_timestamp
-            && node_status
-                .latest_onchain_wallet_sync_timestamp
-                .unwrap_or_default()
-                > esplora_chain_tip_timestamp;
-
         Ok(GetNodeInfoResponse {
             pub_key: self.node.node_id().serialize().to_vec(),
             alias: match self.node.node_alias() {
                 Some(alias) => alias.to_string(),
                 None => format!("LDK Fedimint Gateway Node {}", self.node.node_id()),
             },
             network: self.node.config().network.to_string(),
-            block_height,
-            synced_to_chain,
+            block_height: node_status.current_best_block.height,
+            synced_to_chain: node_status.is_running && node_status.is_listening,
```

Comment:
Does this mean it is synced to the chain? I would have thought you should check LDK's latest block vs the esplora's latest block?